database/1-init.sql

```diff
@@ -0,0 +1,100 @@
+-- 1728826462-init.sql
+
+-- Create the database (if not exists)
+CREATE DATABASE IF NOT EXISTS quizzard;
+
+-- Use the database
+USE quizzard;
+
+-- Create the users table
+CREATE TABLE IF NOT EXISTS users (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    username VARCHAR(50) NOT NULL UNIQUE,
+    name VARCHAR(100) NOT NULL,
+    email VARCHAR(255) NOT NULL UNIQUE,
+    password VARCHAR(255) NOT NULL,
+    profile_picture TEXT,
+    english_proficiency INT NOT NULL,
+    points INT DEFAULT 0,
+    created_quizzes INT DEFAULT 0,
```

Comment:
This field keeps redundant information. We should not forget adding mechanism to keep it sync. Should we leave it to backend or implement a database trigger?